src/plugin/fieldViews/ProseMirrorFieldView.ts

```diff
@@ -51,7 +51,7 @@ export abstract class ProseMirrorFieldView implements FieldView<string> {
     // The ProseMirror node type name
     private readonly fieldName: string,
     // Plugins that the editor should use
-    private plugins?: Plugin[]
+    plugins?: Plugin[]
```

Comment:
Whilst exploring, I noticed this isn't used outside of the constructor, so I tidied it up.